### PR TITLE
organizations claim

### PR DESCRIPTION
**Pull request by kikumar552**

### CODE REVIEW

File: internal/provider/provider_test.go [hunk 1]
```diff
@@ -22,7 +22,7 @@ func testAccPreCheck(t *testing.T) {
 	// function.
 
 	// Check environmental variables
-	ev := []string{"MERAKI_DASHBOARD_API_KEY"}
+	ev := []string{"MERAKI_DASHBOARD_API_KEY, TF_ACC_MERAKI_MX_LICENCE, TF_ACC_MERAKI_MX_SERIAL, TF_ACC_MERAKI_ORDER_NUMBER"}
```
Comment (iamdexterpark) [inline internal/provider/provider_test.go:25]: Seems like the same change as in the add devices resource so this should hopefully not cause a merge conflict.

File: internal/provider/organizations_claim_resource_test.go [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+		]
+	
+	}
+`, order, serial, licence)
+	return string(result)
+}
+
+// testAccOrganizationsClaimResourceConfigUpdate is a constant string that defines the configuration for updating a organizations_claim resource in your tests.
+// It depends on both the organization and network resources.
+func testAccOrganizationsClaimResourceConfigUpdate(order, serial, licence string) string {
```
Comment (iamdexterpark) [inline internal/provider/organizations_claim_resource_test.go:105]: There isn't a need for a Update test as this resource only has the CREATE ability.

File: internal/provider/organizations_claim_resource_test.go [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+const testAccOrganizationsClaimResourceConfigCreateOrganization = `
+ resource "meraki_organization" "test" {
+ 	name = "test_meraki_organizations_claim"
+ 	api_enabled = true
+ }
+ `
+
+// testAccOrganizationsClaimResourceConfigCreate is a constant string that defines the configuration for creating and reading a organizations_claim resource in your tests.
+// It depends on both the organization and network resources.
+func testAccOrganizationsClaimResourceConfigCreate(order, serial, licence string) string {
```
Comment (iamdexterpark) [inline internal/provider/organizations_claim_resource_test.go:83]: Once a license, serial, or order number is claimed there is no way to manually remove it so this is a test which can only be ran a single time. 

Possibly you can move a lic or serial between orgs (DO_NOT_DELETE) however that is a separate resource and out of scope here so we will need to think about ways to better run this test or comment it out completely.

**Review by iamdexterpark — COMMENTED** [review]
Please address the acc_test issues and add the secrets to the repo.

File: internal/provider/organizations_claim_resource_test.go (hunk 3 above)
Comment (iamdexterpark) [reply internal/provider/organizations_claim_resource_test.go:83]: So it turns out that it is possible to unclaim a list of serials at the org level: {{baseUrl}}/organizations/:organizationId/inventory/release

I recommend adding this as part of a for loop to remove any serials in the list on the DELETE call.

File: internal/provider/organizations_claim_resource_test.go (hunk 2 above)
Comment (kikumar552) [reply internal/provider/organizations_claim_resource_test.go]: update test case removed

File: internal/provider/organizations_claim_resource_test.go (hunk 3 above)
Comment (kikumar552) [reply internal/provider/organizations_claim_resource_test.go:64]: added this {{baseUrl}}/organizations/:organizationId/inventory/release api call in delete call

File: internal/provider/provider_test.go (hunk 1 above)
Comment (kikumar552) [reply internal/provider/provider_test.go:25]: added them to secrets

**Review by iamdexterpark — APPROVED** [review]
LGTM

**Pull request merged**